website/docs/advanced/docker/images.md

```diff
@@ -10,21 +10,30 @@ import TabItem from '@theme/TabItem';
 :::info
 This page provides a detailed explanation of Docker 
 images and their hierarchy. For information on the process 
-of building these images, please refer to [this section](/docs/advanced/docker/deploy).
+of building these images, please refer to the 
+[automated](/docs/advanced/docker/automated) or 
+[manual](/docs/advanced/docker/manual) builds sections.
 :::
 
 
-The tools, scripts, dependencies, and configurations utilized by the 
-GATK-SV pipeline, written in WDL, are organized into separate Docker 
-containers. This modular approach ensures that each container 
-contains only the necessary tools for its specific task, 
-resulting in smaller image sizes. This design choice simplifies 
-the definition of Dockerfiles and facilitates easier maintenance. 
-Moreover, the smaller image sizes contribute to reduced disk 
-usage and lower workflow execution costs.
+GATK-SV organizes the tools, scripts, and their dependencies and configurations 
+into multiple Docker images. Each Docker image is built for a specific purpose, 
+and images have a hierarchical dependency. This modular design has several advantages; 
+first, it results in focused and more straightforward instructions in Dockerfiles, 
+facilitating their development, maintenance, and extensibility. 
+Second, it results in smaller Docker images, as each image contains only 
+the related tools and scripts. Smaller images reduce storage costs on container 
+registries and are transferred faster to virtual machines, resulting in shorter start-up. 
+Third, the modular design reduces duplication in Dockerfiles and ensures configuration 
+consistency across different Docker images.
+Finally, this architecture significantly lowers the maintenance cost as it 
+necessitates updating only the affected Docker images throughout the development
+(discussed in details in the [following section](#incremental)). 
```

Comment:
Make this into a list to make it a little more digestable

Reply:
Agreed, that would be more intuitive.